<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>562</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>569</width>
    <height>562</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>569</width>
    <height>562</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>546</width>
      <height>130</height>
     </rect>
    </property>
    <property name="title">
     <string>Registro</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="5">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Precio:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Categoria:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="nombre_lineEdit"/>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="marca_lineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QDoubleSpinBox" name="precio_doubleSpinBox">
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Cantidad:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5" colspan="2">
      <widget class="QSpinBox" name="cantidad_spinBox">
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QLineEdit" name="categoria_lineEdit"/>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Marca:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="7">
      <widget class="QPushButton" name="registrar_pushButton">
       <property name="text">
        <string>Registrar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>551</width>
      <height>367</height>
     </rect>
    </property>
    <property name="title">
     <string>Tabla</string>
    </property>
    <widget class="QTableWidget" name="registros_tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>531</width>
       <height>291</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>Codigo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Marca</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Precio</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Categoria</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cantidad</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="limpiar_pushButton">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>320</y>
       <width>511</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Limpiar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGuardar">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionGuardar"/>
    <addaction name="actionCargar"/>
   </widget>
   <addaction name="menuGuardar"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCargar">
   <property name="text">
    <string>Cargar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
